frontend/src/stock/StockOverview.tsx

```diff
@@ -105,10 +104,14 @@ function StockOverview() {
                                     <td>{singleItem.pricePerKilo}</td>
                                     <td>Lorem</td>
                                     <td>Lorem</td>
-                                    <td className={"action-cell"}><EditIcon onClickAction={openEditModal}
-                                                                            itemToEdit={singleItem}/>
-                                        <DeleteIcon onClickAction={openDeleteModal}
-                                                    idToDelete={singleItem.id}/></td>
+                                    <td>
+                                        <div className={"action-cell"}>
```

Comment:
Why do you need that div? You could add the className to the \<td> fragment

Reply:
That made the height of the td different than the other td before. That's why I had to add the div so I can keep the diplaying as flex